src/Akka.Persistence.Sql.Linq2Db/snapshot.conf

```diff
@@ -30,62 +31,90 @@
       dao = "Akka.Persistence.Sql.Linq2Db.Snapshot.ByteArraySnapshotDao, Akka.Persistence.Sql.Linq2Db"
       
       compatibility-mode = false
+        
+      #if true, tables will attempt to be created.
+      auto-initialize = false
+      
+      #if true, a warning will be logged
+      #if auto-init of tables fails.
+      #set to false if you don't want this warning logged
+      #perhaps if running CI tests or similar.
+      warn-on-auto-init-fail = true
       
-      tables.snapshot {
+      default {
```

Comment:
Same idea as the journal - got it (provided that my assumptions in my comment on the journal are correct - please let me know @Arkatufus )